static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-banner.js

```diff
@@ -24,15 +24,15 @@ define([
     var EditionTest = function (edition) {
 
         this.edition = edition;
-        this.id = 'MembershipEngagementBanner'+edition[0].toUpperCase() + edition.substr(1);
-        this.start = '2016-09-08';
-        this.expiry = '2016-10-08';
+        this.id = 'MembershipEngagementBannerExtended'+edition[0].toUpperCase() + edition.substr(1);
```

Comment:
Per [the docs](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md#writing-a-test), you'll need to update the test name in `ABTestSwitches` to reflect this change in name.

> id: The unique name of the test. This must be TitleCased and correspond to the hyphen-separated portion of the switch id that follows ab-. e.g. if the switch id is ab-geo-most-popular, this id must be GeoMostPopular